.github/workflows/cronjob.yaml

```diff
@@ -0,0 +1,227 @@
+name: Cron job for master&&develop branch
+
+on:
+  schedule:
+    - cron: '0 23 * * *'
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
+  CARGO_TARGET_DIR: ${{ github.workspace }}/../target
+  LOGBAK_USER: ${{secrets.LOGBAK_USER}}  #LOCBAK_* for upload logs to server when test failed, for windows
+  LOGBAK_PASSWORD: ${{secrets.LOGBAK_PASSWORD}}
+  LOGBAK_SERVER: ${{secrets.LOGBAK_SERVER}}
+jobs:
+#job for checkout the ckb
+  checkout:
+    name: ckb-checkout
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [Linux-runner, mac-runner,win-runner]
+        include:
+          - os: Linux-runner
+            script_run: make build
+          - os:  mac-runner
+            script_run: make build
+          - os: win-runner
+            script_run: devtools/windows/make prod
+    steps:
+    - name: Set Cargo path
+      run: echo "$HOME/.cargo/bin" >> $GITHUB_PATH
+    - uses: actions/checkout@v2
+      with:
+        submodules: false
+        fetch-depth: 2
```

Comment:
can we use default parameters?

Reply:
```shell
grep -r submodules . | wc -l
5
```

Reply:
I think it's better to keep the same config with travis.